cfme/web_ui/paginator.py

```diff
@@ -16,6 +16,9 @@
 _page_cell = '//td//td[contains(., " of ")]|//li//span[contains(., " of ")]'
 _check_all = Input("masterToggle")
 
+_regexp = {version.LOWEST: r"(?:Item|Items)*\((\d+).*of (\d+)\)(?:item|items?)*",
+           "5.5": r"Showing (\d+)-(?:\d+) of (\d+) (?:item|items?)?"}
```

Comment:
```python
r"Showing (?P<first>\d+)-(?:\d+) of (?P<total>\d+) (?:item|items?)?"
```

so that later we can do

```python
match.groupdict()['first']
```

gives a nice correlation then.